os/arch/Kconfig

```diff
@@ -66,10 +89,35 @@ config ARCH_NEED_ADDRENV_MAPPING
 	bool
 	default n
 
+config ARCH_HAVE_EXTRA_HEAPS
+        bool
+        default n
+        ---help---
+                Special memory regions used as separate heaps
+
+config ARCH_HAVE_TEXT_HEAP
+        bool
+        default n
+        ---help---
+                Special memory region for dynamic code loading
+
+config ARCH_HAVE_MULTICPU
+        bool
+        default n
+
 config ARCH_HAVE_VFORK
 	bool
 	default n
 
+config ARCH_HAVE_FPU
+        bool
+        default n
+
+config ARCH_HAVE_DPFPU
+        bool
+        default n
+        select ARCH_HAVE_FPU
+
```

Comment:
Looks like these are duplicated, please check and remove
https://github.com/Samsung/TizenRT/blob/a93cd711656fecb66f98b34f9f251c53d1894a49/os/arch/arm/Kconfig#L210-L235